blueprints/app/files/app/deprecation-workflow.ts

```diff
@@ -0,0 +1,25 @@
+import setupDeprecationWorkflow from 'ember-cli-deprecation-workflow';
+
+/**
+ * Docs: https://github.com/ember-cli/ember-cli-deprecation-workflow
+ */
+setupDeprecationWorkflow({
+/**
+    false by default, but if a developer / team wants to be more aggressive about being proactive with
+    handling their deprecations, this should be set to "true"
+*/
+  throwOnUnhandled: false,
+  handlers: [
```

Comment:
```suggestion
  workflow: [
```

`handlers` does not exist.

Reply:
The `ember-cli-deprecation-workflow` docs could use some adjustments I think, weird that there's no code example in the docs that covers this.